### DISCUSSION TITLE
请问使用mc客户端添加子用户报错，已经确认使用的juicefs版本支持了多用户，能帮忙看下么

### DISCUSSION THREAD

**Discussion by yaliJU** [juicedata/juicefs#5089] [Q&A]
背景：公司目前想把ossfs替换成juicefs，调研到目前juicefs s3 gateway高级功能已经支持身份和访问控制，想要验证下juicefs 是否能够 达到同 oss 一样不同用户数据访问隔离的效果。但是目前在给子账户设置读写权限时报错，希望能得到大家的帮助，谢谢！

```
juicefs --version
juicefs version 1.2.0+2024-06-18.873c47b
mc admin policy attach  juicefs readwrite --user 
mc: <ERROR> Unable to make user/group policy association. This 'admin' API is not supported by server in 'mode-server-fs'.

```

超级用户下的bucket和对象列表
<img width="1440" alt="admin" src="https://github.com/user-attachments/assets/49c7a1cc-ca52-4b5c-9b57-4baf8e0b8180">

在超级用户下创建子账号 user1
```
mc admin user add juicefs user1 admin123
```
登陆页面查看到没有任何数据，于是想要通过设置user1权限为readwrite，但是设置失败，下面是user1的登陆后查看的页面
![user1](https://github.com/user-attachments/assets/7087a0f5-db70-4f50-b698-e1b137263c1e)

PS: 还有个疑惑，juicefs是否能够设置类似oss 用户体系，除了支持子账户，还能独立创建不同的用户，达到数据隔离的效果，不然感觉现在的子账户并不能完全支持替换ossfs的账户体系。

**Comment by zhijian-pro** [comment]
mc 用的客户端版本应该不对，看下这个[文档](https://juicefs.com/docs/zh/community/guide/gateway#%E4%BD%BF%E7%94%A8-minio-%E5%AE%A2%E6%88%B7%E7%AB%AF)
